modules/nf-core/fastk/fastk/main.nf

```diff
@@ -26,11 +26,13 @@ process FASTK_FASTK {
     def prefix = task.ext.prefix ?: "${meta.id}"
     def FASTK_VERSION = 'f18a4e6d2207539f7b84461daebc54530a9559b0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
     """
+    mkdir tmp
     FastK \\
         $args \\
         -T$task.cpus \\
         -M${task.memory.toGiga()} \\
         -N${prefix}_fk \\
+        -Ptmp \\
```

Comment:
is it right that there is no space? Does just `.` work, so nextflow assigns the proper temp directory?

Reply:
The original bug report was mentioning there were hidden files cluttering up the work directory, but I'm not sure specifically what these are. The tool creates some hidden files that are specifically needed for the output channels and these remain in the current directory and not in `tmp`. That folder just stays empty.